cmd/completion.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"errors"
+	"os"
+)
+
+// bashCompletionCmd represents the completion command
+var completionCmd = &cobra.Command{
+	Use:       "completion [shell]",
+	Short:     "Output shell completion code for the specified shell (bash or zsh)",
+	Long:      "Output shell completion code for the specified shell (bash or zsh)",
```

Comment:
Please remove `Long` as it's identical to `Short`. Long help will get generated from a reference manual eventually (#158).